go-controller/pkg/node/secondary_node_network_controller_test.go

```diff
@@ -459,7 +459,9 @@ var _ = Describe("SecondaryNodeNetworkController: UserDefinedPrimaryNetwork Gate
 					udnRules = append(udnRules, rule)
 				}
 			}
-			Expect(udnRules).To(HaveLen(3))
+
+			// 3 rules per ip family: see constructUDNVRFIPRules for more details
+			Expect(udnRules).To(HaveLen(6))
```

Comment:
For anyone thinking: we only added 2 new rules, why is it +3 in the test?
It is because we never actually were adding ipv6 pktMark rule before (due to `areNetlinkRulesEqual` bug that is fixed now) and expected rule number shouldn've been 4 before this change (2 for ipv4 and 2 for ipv6 makes sense right?)